site/hi/index.html

```diff
@@ -0,0 +1,669 @@
+			कृपया देखें <a href="https://github.com/eclipse/eclipse-collections/blob/master/CONTRIBUTING.md">कैसे योगदान</a>करनेका आरंभ करें.
+                    </p>
+                </div>
+                <div class="col-lg-4 col-sm-4">
+                    <h3>ताज़ा खबर</h3>
+                    <p class="regular">एक्लिप्स कलेक्शन 10.1.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/10.1.0">प्रकाशित</a> <a class="light">(4 दिसंबर 2019)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 10.0.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/10.0.0">प्रकाशित</a> <a class="light">(10 जुलाई 2019)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 9.2.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/9.2.0">प्रकाशित</a> <a class="light">(20 मई 2018)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 9.1.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/9.1.0">प्रकाशित</a> <a class="light">(1 जनवरी 2018)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 9.0.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/9.0.0">प्रकाशित</a> <a class="light">(21 सितंबर 2017)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 8.2.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/8.2.0">प्रकाशित</a> <a class="light">(22 जून 2017)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 8.1.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/8.1.0">प्रकाशित</a> <a class="light">(16 मार्च 2017)</a></p>
+                    <p class="regular">एक्लिप्स कलेक्शन 8.0.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/8.0.0">प्रकाशित</a> <a class="light">(8 सितंबर 2016)</a></p>
+                </div>
+                <div class="col-lg-4 col-sm-4">
+                    <h3>सामुदायिक लिंक</h3>
+                        <p class="regular">एक्लिप्स पीएमआई: <a href="https://projects.eclipse.org/projects/technology.collections">एक्लिप्स परियोजना प्रबंधन आधारिक संरचना</a></p>
+                        <p class="regular">गिटहब: <a href="https://github.com/eclipse/eclipse-collections">एक्लिप्स-कलेक्शन</a></p>
+                        <p class="regular">मुद्दे: <a href="https://github.com/eclipse/eclipse-collections/issues">गिटहब मुद्दे</a></p>
+                        <p class="regular">ईमेल की सूची: <a href="https://dev.eclipse.org/mailman/listinfo/collections-dev">कलेक्शन-देव</a> मेलिंग सूची</p>
+                        <p class="regular">विकी: <a href="https://github.com/eclipse/eclipse-collections/wiki">गिटहब विकि</a></p>
+                        <p class="regular">मंच: <a href="https://www.eclipse.org/forums/index.php?t=thread&amp;frm_id=329">एक्लिप्स कलेक्शन मंच</a></p>
+                        <p class="regular">स्टाकॉवेर्फ्लो: टैग के साथ <a href="http://stackoverflow.com/questions/tagged/eclipse-collections">एक्लिप्स-कलेक्शन</a></p>
```

Comment:
I think stackoverflow has a typo in it.

Reply:
it reads as "stalk overflow". Use the double-aiy matra

Reply:
fixed